apis/fluentd/v1alpha1/plugins/input/types.go

```diff
@@ -31,6 +31,8 @@ type Input struct {
 	Tail *Tail `json:"tail,omitempty"`
 	// in_sample plugin
 	Sample *Sample `json:"sample,omitempty"`
+	// monitor_agent plugin
+	MonitorAgent *MonitorAgent `json:"monitor_agent,omitempty"`
```

Comment:
camelCase here too: monitor_agent => monitorAgent

Reply:
Oops - missed that one.  Fixed in [1a33bbc](https://github.com/fluent/fluent-operator/pull/967/commits/1a33bbca6b0c1fc39d206e0d3e779eb4395f7dc2).